test/cpp/interop/interop_client.cc

```diff
@@ -451,7 +451,7 @@ bool InteropClient::DoResponseStreaming() {
     // most likely due to connection failure.
     gpr_log(GPR_ERROR,
             "DoResponseStreaming(): Read fewer streams (%d) than "
-            "response_stream_sizes.size() (%" PRIuPTR ")",
+            "response_stream_sizes.size() (%zu)",
```

Comment:
Ugh... I see the state of format specifiers is a mess... it would be `%Iu` for windows. Using `PRIuPTR` works, but it's not the intended use of the macro. I guess it's the best solution given the options... I'm reverting this change.